### ISSUE TITLE
Series title and text alignment error with 'Use light style' option

### ISSUE THREAD

**Issue by mtlemissova** [publishpress/publishpress-series#600]
Your issue may already be reported!
Please search on the [issue track](../) before creating one.

## Expected Behavior

Series title and text should be properly aligned on the website page
## Current Behavior

Series title and text are too close to the left margin of the website

## Steps to Reproduce (for bugs)

1. Series
2. Settings
3. Under 'Display on single posts in a series' area in 'Style Options' select 'Use light style' radio button
4. Under 'Display on Series Table of Contents screens' select '1' radio button
5. Press 'Update Options'
6. View Site
Note: Series title and text alignment is too close to the left margin of the website

## Your Environment

* Version used: Version 2.9.2
* Browser Name and version:  Chrome/Version 104.0.5112.101
* Operating System and version (desktop or mobile): MacOS Monterey / Version 12.4
* Link to your project:
<img width="722" alt="Screen Shot 2022-09-02 at 3 48 05 PM" src="https://user-images.githubusercontent.com/107439940/188229623-aebb28b9-faa8-48f1-9eff-dce57df2026f.png">

**Comment by olatechpro** [comment]
@stevejburge this problem can be solve by moving series to shortcode instead of template as padding and margin varies by  theme.

We can't decide to add general contain class since some theme will have it by default thereby affecting their content.

**Comment by stevejburge** [comment]
Yes, this might be necessary. I couldn't see the original reported problem, but certainly the "Table of Contents" should be moved. 

It is a strange choice to have it floating:

![Screen Shot 2022-09-14 at 1 49 22 PM](https://user-images.githubusercontent.com/3868028/190227473-3651068f-169f-42f2-88ff-f8cdaa4f1038.png)

This leads to some bad layouts:

![Screen Shot 2022-09-14 at 1 53 10 PM](https://user-images.githubusercontent.com/3868028/190227519-d6f72682-c578-4026-9ebe-a273748a83c2.png)

**Comment by olatechpro** [comment]
@stevejburge I’m not sure this issue is related to series table of content but series list page taking full width without leaving left and right margin.

if there’s another request on series table of content, maybe it should move to another issue for discussion?

**Comment by olatechpro** [comment]
Okay, checking again, it seems to be series TOC but with your explanation, I don’t know what you’re proposing.

Are you simply supporting creating shortcode for series TOC as alternative to default templates?

@stevejburge

**Comment by stevejburge** [comment]
Thanks @olatechpro. I opened https://github.com/publishpress/publishpress-series/issues/618

**Comment by mtlemissova** [comment]
Yes, try any default theme and you have the same issue.

Thanks

**Comment by olatechpro** [comment]
closing for https://github.com/publishpress/publishpress-series/issues/618

**Comment by olatechpro** [comment]
@stevejburge do you mean this issue should be closed since you opened new one?

**Comment by olatechpro** [comment]
@stevejburge So, what needed to be done here?

**Comment by stevejburge** [comment]
@olatechpro Let's skip this one for now and see how it's impacted by the changes in 2.10.0
